src/js/edu-benefits/pages/directDepositChange.js

```diff
@@ -25,6 +25,15 @@ export default function createDirectDepositChangePage(schema) {
       bankAccount: _.merge(bankAccountUI, {
         'ui:options': {
           hideIf: (formData) => formData.bankAccountChange !== 'startUpdate'
+        },
+        accountType: {
+          'ui:required': (formData) => _.get('bankAccountChange', formData) === 'startUpdate'
```

Comment:
Great job and good question, sounds like it might make a good candidate for a member of utils/helpers.